Sources/App/main.swift

```diff
@@ -1,13 +1,20 @@
-import Vapor
+import AppLogic
```

Comment:
i was thinking executable originally but that kind of has a "win32" feel to it. Selecting the "Run" scheme in the Xcode project seems extra nice as well.